### PR TITLE
Updates corpora comparison sectionto reflect new additions

### PR DESCRIPTION
**Pull request by danich1**
This PR updates the corpora comparison section to reflect the new additions to the analysis and removes an outdated citations for this section.

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -23,15 +23,16 @@ Out of these 5 million articles, about 3 million were open access and available
...
 
-### Comparing Corpora
-We used gensim [@raw:rehurek_lrec] (version 3.8.1) to preprocess the bioRxiv and PubMed Central corpora.
-We removed the 337 gensim-provided stopwords.
-Throughout our analysis we encountered non-word symbols (e.g., $\pm$), so we refer to words and symbols as tokens to avoid confusion.
+### Comparing Corpora
+We compared bioRxiv against Pubmed Central's Open Access corpus (PMCOA) and the New York Times Annotated corpus (NYTAC) [@raw:sandhaus2008new] to assess the similarities and differences between bioRxiv, PMCOA and NYTAC.
+Throughout our analysis we encountered non-word symbols (e.g., $\pm$), so we refer words and symbols as tokens to avoid confusion.
+We calculated the following statsitics for each corpus: the number of documents, the number of sentences, the total number of tokens, the number of stopwords, the average length of a document, the average length of a sentence, the number of negations, the number of coordinating conjunctions, the number of pronouns and the number of past tense verbs.
```
Comment (cgreene) [inline content/03.methods.md:29]:
```suggestion
We calculated the following statistics for each corpus: the number of documents, the number of sentences, the total number of tokens, the number of stopwords, the average length of a document, the average length of a sentence, the number of negations, the number of coordinating conjunctions, the number of pronouns and the number of past tense verbs.
```

File: content/03.methods.md [hunk 2]
```diff
@@ -23,15 +23,16 @@ Out of these 5 million articles, about 3 million were open access and available
...
+Next, we used spaCy's "en_core_web_sm" model [@raw:spacy2] (version 2.2.3) to preprocess all corpora and filtered out 326 spaCy-provided stopwords.
```
Comment (cgreene) [inline content/03.methods.md:30]: Are these the same stopwords you use in the previous sentence?

File: content/03.methods.md [hunk 3]
```diff
@@ -23,15 +23,16 @@ Out of these 5 million articles, about 3 million were open access and available
...
+Next, we used spaCy's "en_core_web_sm" model [@raw:spacy2] (version 2.2.3) to preprocess all corpora and filtered out 326 spaCy-provided stopwords.  
 
-Following the cleaning process, we calculated the frequency of every token shared between both corpora.
+Following the cleaning process, we calculated the frequency of every token across all corpora.
 Because many tokens were unique to one set or the other and observed at low frequency, we used the union of the top 100 most frequent tokens from each corpus to compare them.
-We generated a contingency table and calculated the odds ratio for each token.
-Furthermore, we also calculated the 95% confidence interval for each odds ratio [@https://www.ncbi.nlm.nih.gov/books/NBK431098/].
+We generated a contingency table for each token and calculated the odds ratio from every generated table.
+We also calculated the 95% confidence interval for each generated odds ratio [@https://www.ncbi.nlm.nih.gov/books/NBK431098/] and measured corpus similarity by calculating the KL divergence across all three corpora.
```
Comment (cgreene) [inline content/03.methods.md:35]:
```suggestion
We also calculated the 95% confidence interval for each token's odds ratio [@https://www.ncbi.nlm.nih.gov/books/NBK431098/] and measured corpus similarity by calculating the KL divergence across all three corpora.
```

**Review by cgreene — APPROVED** [review]
A couple small suggestions and one clarifying question.

File: content/03.methods.md (hunk 2 above)
Comment (danich1) [reply content/03.methods.md:30]: Yes these are the same stopwords that are used by spaCy.

**Pull request merged**